sklearn/preprocessing/_function_transformer.py

```diff
@@ -71,6 +71,17 @@ class FunctionTransformer(TransformerMixin, BaseEstimator):
 
         .. versionadded:: 0.18
 
+    See Also
+    --------
+    MaxAbsScaler : Scale each feature by its maximum absolute value.
+    MinMaxScaler : Transform features by scaling each feature to a given range.
+    QuantileTransformers : Transform features using quantiles information.
+    PowerTransformer : Apply a power transform featurewise to make data
+        more Gaussian-like.
```

Comment:
I would not put the `QuantileTransformer` and the `PowerTransformer`. I think this is enough to have the `MaxAbsScaler`, `StandardScaler`, and for the preprocessor that you added for the labels.